pxc/jenkins/pxc-package-testing-test-yash.groovy

```diff
@@ -0,0 +1,310 @@
+            molecule ${action} -s ${scenario}
+            cd -
+        """
+    }
+}
+
+void setInventories(){
+
+        sh """
+
+            echo \"Setting up Key path based on the selection\"
+
+            if  [[ ${params.TEST_DIST} == "ubuntu-focal" ]]
+            then
+                SSH_USER="ubuntu"            
+                KEYPATH_BOOTSTRAP="/home/ec2-user/.cache/molecule/pxc80-bootstrap/ubuntu-focal/ssh_key-us-west-2"
+                KEYPATH_COMMON="/home/ec2-user/.cache/molecule/pxc80-common/ubuntu-focal/ssh_key-us-west-2"
+            elif [[ ${params.TEST_DIST} == "ubuntu-bionic" ]]
+            then
+                SSH_USER="ubuntu"
+                KEYPATH_BOOTSTRAP="/home/ec2-user/.cache/molecule/pxc80-bootstrap/ubuntu-bionic/ssh_key-us-west-2"
+                KEYPATH_COMMON="/home/ec2-user/.cache/molecule/pxc80-common/ubuntu-bionic/ssh_key-us-west-2"
+            elif [[ ${params.TEST_DIST} == "ubuntu-xenial" ]]
+            then
+                SSH_USER="ubuntu"                
+                KEYPATH_BOOTSTRAP="/home/ec2-user/.cache/molecule/pxc80-bootstrap/ubuntu-xenial/ssh_key-us-west-2"
+                KEYPATH_COMMON="/home/ec2-user/.cache/molecule/pxc80-common/ubuntu-xenial/ssh_key-us-west-2"
+            elif [[ ${params.TEST_DIST} == "debian-11" ]]
+            then
+                SSH_USER="admin"            
+                KEYPATH_BOOTSTRAP="/home/ec2-user/.cache/molecule/pxc80-bootstrap/debian-11/ssh_key-us-west-2"
+                KEYPATH_COMMON="/home/ec2-user/.cache/molecule/pxc80-common/debian-11/ssh_key-us-west-2"
+            elif [[ ${params.TEST_DIST} == "debian-10" ]]
+            then
+                SSH_USER="admin"
+                KEYPATH_BOOTSTRAP="/home/ec2-user/.cache/molecule/pxc80-bootstrap/debian-10/ssh_key-us-west-2"
+                KEYPATH_COMMON="/home/ec2-user/.cache/molecule/pxc80-common/debian-10/ssh_key-us-west-2"
+            elif [[ ${params.TEST_DIST} == "debian-9" ]]
+            then
+                SSH_USER="admin"            
+                KEYPATH_BOOTSTRAP="/home/ec2-user/.cache/molecule/pxc80-bootstrap/debian-9/ssh_key-us-west-2"
+                KEYPATH_COMMON="/home/ec2-user/.cache/molecule/pxc80-common/debian-9/ssh_key-us-west-2"
+            elif [[ ${params.TEST_DIST} == "ol-8" ]]
+            then
+                SSH_USER="ec2-user"
+                KEYPATH_BOOTSTRAP="/home/ec2-user/.cache/molecule/pxc80-bootstrap/ol-8/ssh_key-us-west-2"
+                KEYPATH_COMMON="/home/ec2-user/.cache/molecule/pxc80-common/ol-8/ssh_key-us-west-2"
+            elif [[ ${params.TEST_DIST} == "centos-7" ]]
+            then
+                SSH_USER="centos"
+                KEYPATH_BOOTSTRAP="/home/ec2-user/.cache/molecule/pxc80-bootstrap/centos-7/ssh_key-us-west-2"
+                KEYPATH_COMMON="/home/ec2-user/.cache/molecule/pxc80-common/centos-7/ssh_key-us-west-2"
+            elif [[ ${params.TEST_DIST} == "centos-6" ]]
```

Comment:
centos6 is not supported any more the same as xenial so we should remove it from here.